WordPress/Classes/Services/PageLayoutService.swift

```diff
@@ -1,6 +1,11 @@
 import Foundation
+import Gutenberg
 
 class PageLayoutService {
+    private struct Parameters {
```

Comment:
Are we using a struct here to make it straightforward to add additional parameter keys? I'm not asking because I think this is wrong, I'm just still learning swift and wanted to make sure there wasn't another reason I'm not aware of. 🙂

Reply:
Yup that's it exactly 💯 😃